labels/labels.go

```diff
@@ -26,6 +37,20 @@ var (
 	// ErrReservedPrefix is returned when a label contains the prefix
 	// which is reserved for internally produced labels.
 	ErrReservedPrefix = errors.New("label contains reserved prefix")
+
+	// loopOutSweepSuccess is the label used for loop out swaps to sweep
+	// the HTLC in the success case.
+	LoopOutSweepSuccess = fmt.Sprintf(
```

Comment:
We have two kinds of labels overlapping here - labels for swaps and labels for individual transactions. 
* For swap labels, we use the reserved prefix to ensure that people don't label manual swaps as autoloops (which would mess up our budget). 
* For lnd labels, we do have internal labels (such as channel opens), but do not have a reserved prefix (perhaps we should have, but I didn't think of the requirement at the time, and if people want to intentinally mess up our labelling it's not _really_ our problem, vs autoloop where it could cause actual issues). 

So tldr, I don't think we need the reserved label on these?

Reply:
Ah, I missed the different purposes of the labels. I moved the `lnd` level labels to its own file and added a more verbose pattern that contains some meta information as well.